src/tests/ftest/launch.py

```diff
@@ -7,6 +7,7 @@
 # pylint: disable=too-many-lines
 
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from asyncore import write
```

Comment:
(pylint-deprecated-module) Uses of a deprecated module 'asyncore'
(pylint-unused-import) Unused write imported from asyncore